web/modules/custom/asset_distribution/src/Plugin/Field/FieldWidget/FileSizeCalculatorWidget.php

```diff
@@ -30,8 +30,14 @@ class FileSizeCalculatorWidget extends WidgetBase implements ContainerFactoryPlu
   /**
    * {@inheritdoc}
    */
-  public function __construct($plugin_id, $plugin_definition, FieldDefinitionInterface $field_definition, array $settings, array $third_party_settings,
```

Comment:
A constructor can never inherit the documentation from the parent, since it is constructing a different class.